<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="AGC, AY" />


<title>D62 Preprocessing_Countdata</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DEPDC5_D62_Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">D62 Preprocessing_Countdata</h1>
<h4 class="author">AGC, AY</h4>
<h4 class="date">15 3 2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-10-11
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>DEPDC5_D62_Analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220808code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220808)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220808code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220808)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomKJPMolgenLabDEPDC5D62Analysistree9adf10f86d6c69214abcd0ec546ebb2a6ed33614targetblank9adf10fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/KJPMolgenLab/DEPDC5_D62_Analysis/tree/9adf10f86d6c69214abcd0ec546ebb2a6ed33614" target="_blank">9adf10f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomKJPMolgenLabDEPDC5D62Analysistree9adf10f86d6c69214abcd0ec546ebb2a6ed33614targetblank9adf10fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/KJPMolgenLab/DEPDC5_D62_Analysis/tree/9adf10f86d6c69214abcd0ec546ebb2a6ed33614" target="_blank">9adf10f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  data/.~lock.D62_Sample_info_CePTER_RNASeq.csv#

Unstaged changes:
    Modified:   _workflowr.yml
    Modified:   analysis/02_D62_Preprocessing_counts.Rmd
    Modified:   analysis/MDSplots/D62_mdsplots.RData
    Modified:   output/D62_dds_matrix.RData
    Modified:   wflow_helper.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with
<code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<pre class="r"><code>home = getwd()</code></pre>
<div id="preprocessing" class="section level2">
<h2>Preprocessing</h2>
<pre class="r"><code>filetarget= paste0(home,&quot;/data/Countmatrix.RData&quot;)
load(filetarget)


Ntot= nrow(Countdata)


#merge non unique annotations
if(length(unique(rownames(Countdata))) != Ntot){
  Countdata = Countdata %&gt;% group_by(row.names(Countdata)) %&gt;% summarise_each(sum)
  Ntot= nrow(Countdata)
}


hgnc=gconvert(query=as.numeric(rownames(Countdata)), 
              organism = &quot;hsapiens&quot;, 
              numeric_ns = &quot;ENTREZGENE_ACC&quot;,
              target = &quot;HGNC&quot;)

Ids = hgnc %&gt;%  dplyr::select(name, input, description) %&gt;% group_by(input) %&gt;% 
  summarise(name=paste(name, sep=&quot;; &quot;, collapse = &quot;;&quot;), description = dplyr::first(description))

rowdescription = data.frame(entrez_gene = Ids$input, 
                            hgnc=Ids$name, 
                            description=Ids$description)


rowdescription = rowdescription[match(row.names(Countdata), rowdescription$entrez_gene),]
rownames(rowdescription)=row.names(Countdata)


# load and parse sample information 
SampleInfo=read.csv2(paste0(home,&quot;/data/D62_Sample_info_CePTER_RNASeq.csv&quot;), 
                     row.names = 1)

SampleInfo$Row=gsub(&quot;[0-9]*&quot;,&quot;&quot;,SampleInfo$Position)
SampleInfo$Col=as.numeric(gsub(&quot;[A-Z]*&quot;,&quot;&quot;,SampleInfo$Position))

# set factors and relevel
SampleInfo$CellLine = as.factor(SampleInfo$CellLine)

SampleInfo$gRNA = paste0(&quot;sg&quot;,SampleInfo$gRNA)
SampleInfo$gRNA = factor(SampleInfo$gRNA, levels=c(&quot;sgNTC&quot;, &quot;sg2.1&quot;, &quot;sg2.2&quot;), 
                         labels=c(&quot;sgNTC&quot;, &quot;sg2.1&quot;, &quot;sg2.2&quot;))
SampleInfo$gRNA = relevel(SampleInfo$gRNA,ref=&quot;sgNTC&quot; )

SampleInfo$KO = factor(SampleInfo$KO, levels=c(T,F), labels=c(&quot;KO&quot;, &quot;WT&quot;))
SampleInfo$KO = relevel(SampleInfo$KO,ref=&quot;WT&quot; )

SampleInfo$DIFF = factor(SampleInfo$DIFF, levels=c(TRUE,FALSE), 
                         labels=c(&quot;DIFF&quot;, &quot;noDIFF&quot;))
SampleInfo$DIFF = relevel(SampleInfo$DIFF,ref=&quot;noDIFF&quot;)

SampleInfo$RAPA = factor(SampleInfo$RAPA, levels=c(T,F), 
                         labels=c(&quot;RAPA&quot;, &quot;noRAPA&quot;))

SampleInfo$RAPA = relevel(SampleInfo$RAPA,ref=&quot;noRAPA&quot;)

SampleInfo$label = with(SampleInfo, paste(CellLine,gRNA,DIFF,RAPA, sep=&quot;_&quot;))
SampleInfo$fastQID = rownames(SampleInfo)
SampleInfo = SampleInfo %&gt;% dplyr::group_by(label) %&gt;% mutate(replicate=seq(n())) %&gt;% as.data.frame()
SampleInfo$label_rep=with(SampleInfo, paste(label,replicate,sep=&quot;_&quot;))
rownames(SampleInfo)=SampleInfo$fastQID

# align datasets
checkfiles = all(rownames(SampleInfo) %in% colnames(Countdata))
IDs=intersect(rownames(SampleInfo), colnames(Countdata))
Countdata = Countdata[,IDs]
SampleInfo = SampleInfo[IDs, ]

SampleInfo$reads_per_sample = colSums(Countdata)</code></pre>
<pre class="r"><code>display_tab(head(Countdata))</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:900px; ">
<table class=" lightable-classic" style="font-family: &quot;Arial Narrow&quot;, &quot;Source Sans Pro&quot;, sans-serif; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE10NGSUKBR112901
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE80NGSUKBR112902
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE53NGSUKBR112903
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE26NGSUKBR112904
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE96NGSUKBR112905
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE69NGSUKBR112906
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE42NGSUKBR112907
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE15NGSUKBR112908
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE85NGSUKBR112909
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE58NGSUKBR112910
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE31NGSUKBR112911
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE04NGSUKBR112912
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE74NGSUKBR112913
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE47NGSUKBR112914
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE20NGSUKBR112915
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE90NGSUKBR112916
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE63NGSUKBR112917
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE36NGSUKBR112918
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE09NGSUKBR112919
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE79NGSUKBR112920
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE52NGSUKBR112921
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE25NGSUKBR112922
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE95NGSUKBR112923
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE68NGSUKBR112924
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE41NGSUKBR112925
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE14NGSUKBR112926
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE84NGSUKBR112927
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE57NGSUKBR112928
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE30NGSUKBR112929
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE03NGSUKBR112930
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE73NGSUKBR112931
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE46NGSUKBR112932
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE19NGSUKBR112933
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE89NGSUKBR112934
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE62NGSUKBR112935
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DE35NGSUKBR112936
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
100287102
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
653635
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
82
</td>
<td style="text-align:left;">
133
</td>
<td style="text-align:left;">
121
</td>
<td style="text-align:left;">
84
</td>
<td style="text-align:left;">
78
</td>
<td style="text-align:left;">
31
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
47
</td>
<td style="text-align:left;">
59
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
113
</td>
<td style="text-align:left;">
66
</td>
<td style="text-align:left;">
71
</td>
<td style="text-align:left;">
99
</td>
<td style="text-align:left;">
101
</td>
<td style="text-align:left;">
59
</td>
<td style="text-align:left;">
55
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
77
</td>
<td style="text-align:left;">
42
</td>
<td style="text-align:left;">
27
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
41
</td>
<td style="text-align:left;">
85
</td>
<td style="text-align:left;">
90
</td>
<td style="text-align:left;">
112
</td>
<td style="text-align:left;">
75
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
25
</td>
<td style="text-align:left;">
42
</td>
<td style="text-align:left;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
102466751
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
100302278
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
645520
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
79501
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
<td style="text-align:left;">
0
</td>
</tr>
</tbody>
</table>
</div>
<pre class="r"><code>display_tab(SampleInfo)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:900px; ">
<table class=" lightable-classic" style="font-family: &quot;Arial Narrow&quot;, &quot;Source Sans Pro&quot;, sans-serif; width: auto !important; ">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Plate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Position
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Row
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Col
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
CellLine
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
gRNA
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
KO
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
DIFF
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
RAPA
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Conc
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
UV260_280
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
UV260_230
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
fastQID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
replicate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
label_rep
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
reads_per_sample
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
DE10NGSUKBR112901
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A01
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
0.588
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA
</td>
<td style="text-align:left;">
DE10NGSUKBR112901
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA_1
</td>
<td style="text-align:left;">
8167558
</td>
</tr>
<tr>
<td style="text-align:left;">
DE80NGSUKBR112902
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A02
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
3.6
</td>
<td style="text-align:left;">
1.8
</td>
<td style="text-align:left;">
0.529
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA
</td>
<td style="text-align:left;">
DE80NGSUKBR112902
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA_2
</td>
<td style="text-align:left;">
7947513
</td>
</tr>
<tr>
<td style="text-align:left;">
DE53NGSUKBR112903
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A03
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
0.714
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA
</td>
<td style="text-align:left;">
DE53NGSUKBR112903
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_RAPA_3
</td>
<td style="text-align:left;">
8927353
</td>
</tr>
<tr>
<td style="text-align:left;">
DE26NGSUKBR112904
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A04
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
14.8
</td>
<td style="text-align:left;">
1.682
</td>
<td style="text-align:left;">
1.276
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE26NGSUKBR112904
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA_1
</td>
<td style="text-align:left;">
6192682
</td>
</tr>
<tr>
<td style="text-align:left;">
DE96NGSUKBR112905
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A05
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
10.4
</td>
<td style="text-align:left;">
2.167
</td>
<td style="text-align:left;">
1.529
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE96NGSUKBR112905
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA_2
</td>
<td style="text-align:left;">
6316070
</td>
</tr>
<tr>
<td style="text-align:left;">
DE69NGSUKBR112906
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A06
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
2.22
</td>
<td style="text-align:left;">
1.33
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE69NGSUKBR112906
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sgNTC_DIFF_noRAPA_3
</td>
<td style="text-align:left;">
7211176
</td>
</tr>
<tr>
<td style="text-align:left;">
DE42NGSUKBR112907
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A07
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
1.25
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA
</td>
<td style="text-align:left;">
DE42NGSUKBR112907
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA_1
</td>
<td style="text-align:left;">
6472088
</td>
</tr>
<tr>
<td style="text-align:left;">
DE15NGSUKBR112908
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A08
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
5.2
</td>
<td style="text-align:left;">
1.857
</td>
<td style="text-align:left;">
1.444
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA
</td>
<td style="text-align:left;">
DE15NGSUKBR112908
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA_2
</td>
<td style="text-align:left;">
6381728
</td>
</tr>
<tr>
<td style="text-align:left;">
DE85NGSUKBR112909
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A09
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
5.6
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
1.556
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA
</td>
<td style="text-align:left;">
DE85NGSUKBR112909
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_RAPA_3
</td>
<td style="text-align:left;">
7515594
</td>
</tr>
<tr>
<td style="text-align:left;">
DE58NGSUKBR112910
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A10
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
12.4
</td>
<td style="text-align:left;">
1.938
</td>
<td style="text-align:left;">
1.722
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE58NGSUKBR112910
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA_1
</td>
<td style="text-align:left;">
8072060
</td>
</tr>
<tr>
<td style="text-align:left;">
DE31NGSUKBR112911
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A11
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
13.6
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
0.895
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE31NGSUKBR112911
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA_2
</td>
<td style="text-align:left;">
9132042
</td>
</tr>
<tr>
<td style="text-align:left;">
DE04NGSUKBR112912
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
A12
</td>
<td style="text-align:left;">
A
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
8.4
</td>
<td style="text-align:left;">
1.909
</td>
<td style="text-align:left;">
0.244
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE04NGSUKBR112912
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.1_DIFF_noRAPA_3
</td>
<td style="text-align:left;">
9158749
</td>
</tr>
<tr>
<td style="text-align:left;">
DE74NGSUKBR112913
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B01
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
4.4
</td>
<td style="text-align:left;">
1.833
</td>
<td style="text-align:left;">
1.571
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA
</td>
<td style="text-align:left;">
DE74NGSUKBR112913
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA_1
</td>
<td style="text-align:left;">
8022580
</td>
</tr>
<tr>
<td style="text-align:left;">
DE47NGSUKBR112914
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B02
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
4.4
</td>
<td style="text-align:left;">
1.833
</td>
<td style="text-align:left;">
0.5
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA
</td>
<td style="text-align:left;">
DE47NGSUKBR112914
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA_2
</td>
<td style="text-align:left;">
5290014
</td>
</tr>
<tr>
<td style="text-align:left;">
DE20NGSUKBR112915
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B03
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
1.875
</td>
<td style="text-align:left;">
0.172
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA
</td>
<td style="text-align:left;">
DE20NGSUKBR112915
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_RAPA_3
</td>
<td style="text-align:left;">
7350899
</td>
</tr>
<tr>
<td style="text-align:left;">
DE90NGSUKBR112916
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B04
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
5.2
</td>
<td style="text-align:left;">
1.857
</td>
<td style="text-align:left;">
0.334
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE90NGSUKBR112916
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA_1
</td>
<td style="text-align:left;">
7142950
</td>
</tr>
<tr>
<td style="text-align:left;">
DE63NGSUKBR112917
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B05
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
1.667
</td>
<td style="text-align:left;">
0.789
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE63NGSUKBR112917
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA_2
</td>
<td style="text-align:left;">
6379496
</td>
</tr>
<tr>
<td style="text-align:left;">
DE36NGSUKBR112918
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B06
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
DIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
1.667
</td>
<td style="text-align:left;">
1.25
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA
</td>
<td style="text-align:left;">
DE36NGSUKBR112918
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.2_DIFF_noRAPA_3
</td>
<td style="text-align:left;">
6345821
</td>
</tr>
<tr>
<td style="text-align:left;">
DE09NGSUKBR112919
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B07
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
2.037
</td>
<td style="text-align:left;">
1.25
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE09NGSUKBR112919
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA_1
</td>
<td style="text-align:left;">
6954777
</td>
</tr>
<tr>
<td style="text-align:left;">
DE79NGSUKBR112920
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B08
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
14.8
</td>
<td style="text-align:left;">
2.176
</td>
<td style="text-align:left;">
0.698
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE79NGSUKBR112920
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA_2
</td>
<td style="text-align:left;">
6247879
</td>
</tr>
<tr>
<td style="text-align:left;">
DE52NGSUKBR112921
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B09
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
19.2
</td>
<td style="text-align:left;">
2.087
</td>
<td style="text-align:left;">
1.371
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE52NGSUKBR112921
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_RAPA_3
</td>
<td style="text-align:left;">
7651123
</td>
</tr>
<tr>
<td style="text-align:left;">
DE25NGSUKBR112922
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B10
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
16.8
</td>
<td style="text-align:left;">
1.909
</td>
<td style="text-align:left;">
0.525
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE25NGSUKBR112922
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA_1
</td>
<td style="text-align:left;">
8143934
</td>
</tr>
<tr>
<td style="text-align:left;">
DE95NGSUKBR112923
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B11
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
18.8
</td>
<td style="text-align:left;">
1.958
</td>
<td style="text-align:left;">
1.343
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE95NGSUKBR112923
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA_2
</td>
<td style="text-align:left;">
7710977
</td>
</tr>
<tr>
<td style="text-align:left;">
DE68NGSUKBR112924
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
B12
</td>
<td style="text-align:left;">
B
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sgNTC
</td>
<td style="text-align:left;">
WT
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
17.6
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
1.189
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE68NGSUKBR112924
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sgNTC_noDIFF_noRAPA_3
</td>
<td style="text-align:left;">
9158766
</td>
</tr>
<tr>
<td style="text-align:left;">
DE41NGSUKBR112925
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C01
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
26.8
</td>
<td style="text-align:left;">
2.03
</td>
<td style="text-align:left;">
1.914
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE41NGSUKBR112925
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA_1
</td>
<td style="text-align:left;">
7883862
</td>
</tr>
<tr>
<td style="text-align:left;">
DE14NGSUKBR112926
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C02
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
24.8
</td>
<td style="text-align:left;">
2.067
</td>
<td style="text-align:left;">
1.59
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE14NGSUKBR112926
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA_2
</td>
<td style="text-align:left;">
7134773
</td>
</tr>
<tr>
<td style="text-align:left;">
DE84NGSUKBR112927
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C03
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
20.8
</td>
<td style="text-align:left;">
2.167
</td>
<td style="text-align:left;">
1.268
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE84NGSUKBR112927
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_RAPA_3
</td>
<td style="text-align:left;">
8617208
</td>
</tr>
<tr>
<td style="text-align:left;">
DE57NGSUKBR112928
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C04
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
14.4
</td>
<td style="text-align:left;">
2.571
</td>
<td style="text-align:left;">
0.184
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE57NGSUKBR112928
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA_1
</td>
<td style="text-align:left;">
7544453
</td>
</tr>
<tr>
<td style="text-align:left;">
DE30NGSUKBR112929
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C05
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
16.4
</td>
<td style="text-align:left;">
2.158
</td>
<td style="text-align:left;">
0.911
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE30NGSUKBR112929
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA_2
</td>
<td style="text-align:left;">
7622380
</td>
</tr>
<tr>
<td style="text-align:left;">
DE03NGSUKBR112930
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C06
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.1
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
12.8
</td>
<td style="text-align:left;">
2.286
</td>
<td style="text-align:left;">
0.711
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE03NGSUKBR112930
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.1_noDIFF_noRAPA_3
</td>
<td style="text-align:left;">
7939374
</td>
</tr>
<tr>
<td style="text-align:left;">
DE73NGSUKBR112931
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C07
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
19.6
</td>
<td style="text-align:left;">
1.96
</td>
<td style="text-align:left;">
1.69
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE73NGSUKBR112931
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA_1
</td>
<td style="text-align:left;">
7320431
</td>
</tr>
<tr>
<td style="text-align:left;">
DE46NGSUKBR112932
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C08
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
2.045
</td>
<td style="text-align:left;">
1.607
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE46NGSUKBR112932
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA_2
</td>
<td style="text-align:left;">
6532522
</td>
</tr>
<tr>
<td style="text-align:left;">
DE19NGSUKBR112933
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C09
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
RAPA
</td>
<td style="text-align:left;">
17.6
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
1.63
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA
</td>
<td style="text-align:left;">
DE19NGSUKBR112933
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_RAPA_3
</td>
<td style="text-align:left;">
7115292
</td>
</tr>
<tr>
<td style="text-align:left;">
DE89NGSUKBR112934
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C10
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
15.2
</td>
<td style="text-align:left;">
2.111
</td>
<td style="text-align:left;">
1.583
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE89NGSUKBR112934
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA_1
</td>
<td style="text-align:left;">
7618310
</td>
</tr>
<tr>
<td style="text-align:left;">
DE62NGSUKBR112935
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C11
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
14.4
</td>
<td style="text-align:left;">
2.25
</td>
<td style="text-align:left;">
1.091
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE62NGSUKBR112935
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA_2
</td>
<td style="text-align:left;">
7387684
</td>
</tr>
<tr>
<td style="text-align:left;">
DE35NGSUKBR112936
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
C12
</td>
<td style="text-align:left;">
C
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
D62
</td>
<td style="text-align:left;">
sg2.2
</td>
<td style="text-align:left;">
KO
</td>
<td style="text-align:left;">
noDIFF
</td>
<td style="text-align:left;">
noRAPA
</td>
<td style="text-align:left;">
15.2
</td>
<td style="text-align:left;">
2.111
</td>
<td style="text-align:left;">
0.717
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA
</td>
<td style="text-align:left;">
DE35NGSUKBR112936
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
D62_sg2.2_noDIFF_noRAPA_3
</td>
<td style="text-align:left;">
7440730
</td>
</tr>
</tbody>
</table>
</div>
<p>Total number of samples overlapping between Counts and SampleInfo:
36</p>
<pre class="r"><code>boxplot_counts = function(plotsubset,maintitle,colorcode){
  vals=log2(plotsubset+1)
  a =boxplot(vals, main = maintitle, 
             col = Dark8[as.factor(SampleInfo[,colorcode])], names=NA,
             ylab = &quot;log2 transformed&quot;, xlab=&quot;samples&quot;, xaxt=&quot;n&quot;)
  legend(ncol(vals)*1.1, max(vals), legend = levels(SampleInfo[,colorcode]),
         bg=&quot;white&quot;,xpd=T,box.col = &quot;white&quot;,
         pch = 16, col = Dark8[1:length(unique(SampleInfo[,colorcode]))])
}


barplot_counts = function(DF, maintitle, colorcode) {

  vals=log2(DF[,&quot;reads_per_sample&quot;])
  barplot(vals, main = maintitle, 
          col = Dark8[as.factor(DF[,colorcode])], names=NA, xaxt=&quot;n&quot;,
          ylab = &quot;log2 transformed&quot;, xlab=&quot;samples&quot;)
  legend(length(vals)*1.25, max(vals), legend = levels(DF[,colorcode]), pch = 16, 
         bg =&quot;white&quot;,xpd=T, box.col=&quot;white&quot;,
         col = Dark8[1:length(unique(DF[,colorcode]))])
}


par(mar=c(3,5,5,7))
boxplot_counts(Countdata, &quot;raw counts&quot;, &quot;gRNA&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/plot_counts-1.png" width="4000" style="display: block; margin: auto;" /></p>
<pre class="r"><code>barplot_counts(SampleInfo, &quot;total reads&quot;, &quot;gRNA&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/plot_counts-2.png" width="4000" style="display: block; margin: auto;" /></p>
<pre class="r"><code>plot(density(log2(rowMeans(Countdata))), main=&quot;distribution of gene expression&quot;, 
     xlab=&quot;mean log2(counts +1)&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/plot_counts-3.png" width="4000" style="display: block; margin: auto;" /></p>
<pre class="r"><code># remove genes wich were not detected in at least 50% of the samples 
keeperidx = rowSums(Countdata&gt;1)&gt;nrow(SampleInfo)/2

Countdata_cl = Countdata[keeperidx, ]

rowdescription = rowdescription[row.names(Countdata_cl),]

fullmodel = as.formula(&quot;~gRNA+DIFF+RAPA&quot;)


ddsMat &lt;- DESeqDataSetFromMatrix(countData = Countdata_cl,
                                 colData = SampleInfo,
                                 rowData = rowdescription,
                                 design = fullmodel)


ddsMat = estimateSizeFactors(ddsMat)
ddsMat = estimateDispersions(ddsMat)</code></pre>
<pre><code>gene-wise dispersion estimates</code></pre>
<pre><code>mean-dispersion relationship</code></pre>
<pre><code>final dispersion estimates</code></pre>
<pre class="r"><code>reads  = as.data.frame(counts(ddsMat, normalized=T))

SDs = apply(reads, 1, sd)
keepvar = SDs&gt;0

ddsMat &lt;- ddsMat[keepvar,]

Nfilt = length(ddsMat)
reads  = as.data.frame(counts(ddsMat, normalized=T))

SampleInfo$reads_per_sample_cl= colSums(reads)</code></pre>
<div id="before-cleaning" class="section level3">
<h3>before cleaning</h3>
<ul>
<li>Average reads per samples: 7 445 857</li>
<li>Standard deviation reads per samples: 921 177.8</li>
<li>Total genes mapped: 28 395</li>
</ul>
</div>
<div id="after-cleaning" class="section level3">
<h3>after cleaning</h3>
<ul>
<li>Average reads per samples: 7 195 819</li>
<li>Standard deviation reads per samples: 345 374</li>
<li>Genes removed due to low reads: 14 664</li>
<li>Total genes included after filtering: 13 731</li>
</ul>
</div>
</div>
<div id="clustering" class="section level2">
<h2>Clustering</h2>
<p>hierarchical clustering based on the top 2000 genes by variance</p>
<pre class="r"><code>log2_cpm = log2(reads+1)

varsset=apply(log2_cpm, 1, var)

cpm.sel.trans = t(log2_cpm[order(varsset,decreasing = T)[1:2000],])

rownames(cpm.sel.trans)=SampleInfo$label_rep

distance = dist(cpm.sel.trans)
hc = hclust(distance, method=&quot;ward.D2&quot;)
cutN=12 #number of different conditions (DIFF, RAPA, sgRNA)
clusters = cutree(hc, k=cutN)
Colors=sample(jetcolors(cutN))[clusters]

myLetters &lt;- LETTERS[1:26]

numRow=match(SampleInfo$Row, myLetters)
numRow=numRow+(SampleInfo$Plate-1)*8
addRow=LETTERS[numRow]

Plotdata=data.frame(Rows=addRow, numRow = numRow, Cols = SampleInfo$Col, 
                    Group=clusters, Colors=Colors)

par(mar=c(15,3,5,3))
plot(as.dendrogram(hc), main=paste(&quot;Similairtiy by gene expression, guessed&quot;,cutN,&quot;clusters&quot;), cex=0.7)
colored_dots(colors = Colors, dend = as.dendrogram(hc), rowLabels = &quot;cluster&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/hcluster-1.png" width="7500" style="display: block; margin: auto;" /></p>
<p>Similarity based on hcluster plot</p>
<pre class="r"><code>par(mar=c(2,5,8,3))
plot(0,0, type=&quot;n&quot;, ylab=&quot;&quot;, xlab=&quot;&quot;, 
     ylim=rev(range(Plotdata$numRow))+c(1,-1), 
     xlim=range(Plotdata$Cols)+c(-1,1), xaxt=&quot;n&quot;,yaxt=&quot;n&quot; ,
     main=&quot;plate similarity plot&quot;)
points(y=Plotdata$numRow, x=Plotdata$Cols, pch=16, cex=4, col=Plotdata$Colors)
text(y=Plotdata$numRow, x=Plotdata$Cols, labels = Plotdata$Group)
text(y=Plotdata$numRow, x=Plotdata$Cols, labels = Plotdata$Group)
axis(2, at=1:9, labels = c(paste0(&quot;P1_&quot;, LETTERS[1:8]), &quot;P2_A&quot;), las=1)
axis(3, at=1:12, labels = c(paste0(&quot;Col_&quot;, 1:12)), las=3)
abline(h=8.5)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/platesetup_cluster-1.png" width="7500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>sampleDistMatrix &lt;- as.matrix(distance)

#colors for plotting heatmap
colors &lt;- colorRampPalette(brewer.pal(9, &quot;Spectral&quot;))(255)

cellcol = Dark8[1:nlevels(SampleInfo$CellLine)]
names(cellcol) = levels(SampleInfo$CellLine)

gRNAcol = Dark8[c(1:nlevels(SampleInfo$gRNA))+nlevels(SampleInfo$CellLine)]
names(gRNAcol) = levels(SampleInfo$gRNA)

diffcol = brewer.pal(3,&quot;Set1&quot;)[1:nlevels(SampleInfo$DIFF)]
names(diffcol) = levels(SampleInfo$DIFF)

rapacol = brewer.pal(3,&quot;Set2&quot;)[1:nlevels(SampleInfo$RAPA)]
names(rapacol) = levels(SampleInfo$RAPA)

ann_colors = list(
  DIFF = diffcol, 
  RAPA = rapacol,
  gRNA = gRNAcol
  #,CellLine=cellcol
  )

labels = SampleInfo[,c(&quot;gRNA&quot;,&quot;DIFF&quot;, &quot;RAPA&quot;)] %&gt;%  
  mutate_all(as.character) %&gt;% as.data.frame()

rownames(labels)=SampleInfo$label_rep

pheatmap(sampleDistMatrix,
         clustering_distance_rows = distance,
         clustering_distance_cols = distance,
         clustering_method = &quot;ward.D2&quot;,
         scale =&quot;row&quot;,
         border_color = NA, 
         annotation_row = labels,
         annotation_col = labels,
         annotation_colors = ann_colors,
         col = colors, 
         main = &quot;D62 Distances normalized log2 counts&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/clusterandqual-1.png" width="7500" style="display: block; margin: auto;" /></p>
</div>
<div id="pca-and-mds" class="section level2">
<h2>PCA and MDS</h2>
<pre class="r"><code># PCA
gpca &lt;- glmpca(t(cpm.sel.trans), L = 2)
gpca.dat &lt;- gpca$factors
gpca.dat$CellLine &lt;- SampleInfo$CellLine
gpca.dat$gRNA &lt;- SampleInfo$gRNA
gpca.dat$KO&lt;- SampleInfo$KO
gpca.dat$DIFF &lt;- SampleInfo$DIFF
gpca.dat$RAPA&lt;- SampleInfo$RAPA
gpca.dat$Growth_cond = paste(SampleInfo$DIFF, SampleInfo$RAPA, sep=&quot;_&quot;)

rownames(gpca.dat) = SampleInfo$labels
mds = as.data.frame(SampleInfo) %&gt;% cbind(cmdscale(distance))
mds$Growth_cond = paste(SampleInfo$DIFF, SampleInfo$RAPA, sep=&quot;_&quot;)

save(mds, gpca.dat, file=paste0(home, &quot;/analysis/MDSplots/D62_mdsplots.RData&quot;))

ggplot(gpca.dat, aes(x = dim1, y = dim2, color = gRNA, 
                     shape = Growth_cond)) +
            geom_point(size = 2)  + ggtitle(&quot;PCA with log2 counts D62&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/PCA_MDS-1.png" width="4500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ggplot(mds, aes(x = `1`, y = `2`, color = gRNA, shape = Growth_cond)) +
            geom_point(size = 2)  + ggtitle(&quot;MDS with log2 counts D62&quot;)</code></pre>
<p><img src="figure/02_D62_Preprocessing_counts.Rmd/PCA_MDS-2.png" width="4500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>save(ddsMat, file=paste0(output,&quot;/D62_dds_matrix.RData&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.0 (2022-04-22 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.utf8  LC_CTYPE=German_Germany.utf8   
[3] LC_MONETARY=German_Germany.utf8 LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.utf8    

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] gprofiler2_0.2.1            dendextend_1.16.0          
 [3] glmpca_0.2.0                RCurl_1.98-1.8             
 [5] knitr_1.40                  DESeq2_1.36.0              
 [7] SummarizedExperiment_1.26.1 Biobase_2.56.0             
 [9] MatrixGenerics_1.8.1        matrixStats_0.62.0         
[11] GenomicRanges_1.48.0        GenomeInfoDb_1.32.4        
[13] IRanges_2.30.1              S4Vectors_0.34.0           
[15] BiocGenerics_0.42.0         pheatmap_1.0.12            
[17] RColorBrewer_1.1-3          compareGroups_4.5.1        
[19] forcats_0.5.2               stringr_1.4.1              
[21] dplyr_1.0.10                purrr_0.3.4                
[23] readr_2.1.2                 tidyr_1.2.1                
[25] tibble_3.1.8                ggplot2_3.3.6              
[27] tidyverse_1.3.2             kableExtra_1.3.4           
[29] workflowr_1.7.0            

loaded via a namespace (and not attached):
  [1] readxl_1.4.1           uuid_1.1-0             backports_1.4.1       
  [4] systemfonts_1.0.4      lazyeval_0.2.2         splines_4.2.0         
  [7] BiocParallel_1.30.3    digest_0.6.29          htmltools_0.5.3       
 [10] viridis_0.6.2          fansi_1.0.3            magrittr_2.0.3        
 [13] Rsolnp_1.16            memoise_2.0.1          googlesheets4_1.0.1   
 [16] tzdb_0.3.0             Biostrings_2.64.1      annotate_1.74.0       
 [19] modelr_0.1.9           officer_0.4.4          svglite_2.1.0         
 [22] colorspace_2.0-3       blob_1.2.3             rvest_1.0.3           
 [25] haven_2.5.1            xfun_0.33              callr_3.7.2           
 [28] crayon_1.5.1           jsonlite_1.8.0         genefilter_1.78.0     
 [31] survival_3.4-0         glue_1.6.2             gtable_0.3.1          
 [34] gargle_1.2.1           zlibbioc_1.42.0        XVector_0.36.0        
 [37] webshot_0.5.3          DelayedArray_0.22.0    scales_1.2.1          
 [40] DBI_1.1.3              Rcpp_1.0.9             viridisLite_0.4.1     
 [43] xtable_1.8-4           bit_4.0.4              truncnorm_1.0-8       
 [46] htmlwidgets_1.5.4      httr_1.4.4             ellipsis_0.3.2        
 [49] mice_3.14.0            farver_2.1.1           pkgconfig_2.0.3       
 [52] XML_3.99-0.10          nnet_7.3-17            sass_0.4.2            
 [55] dbplyr_2.2.1           locfit_1.5-9.6         utf8_1.2.2            
 [58] labeling_0.4.2         tidyselect_1.1.2       rlang_1.0.6           
 [61] later_1.3.0            AnnotationDbi_1.58.0   munsell_0.5.0         
 [64] cellranger_1.1.0       tools_4.2.0            cachem_1.0.6          
 [67] cli_3.4.1              generics_0.1.3         RSQLite_2.2.17        
 [70] broom_1.0.1            evaluate_0.16          fastmap_1.1.0         
 [73] yaml_2.3.5             processx_3.7.0         bit64_4.0.5           
 [76] fs_1.5.2               zip_2.2.1              KEGGREST_1.36.3       
 [79] whisker_0.4            xml2_1.3.3             compiler_4.2.0        
 [82] rstudioapi_0.14        plotly_4.10.0          png_0.1-7             
 [85] reprex_2.0.2           geneplotter_1.74.0     bslib_0.4.0           
 [88] stringi_1.7.8          HardyWeinberg_1.7.5    highr_0.9             
 [91] ps_1.7.1               gdtools_0.2.4          lattice_0.20-45       
 [94] Matrix_1.5-1           vctrs_0.4.1            pillar_1.8.1          
 [97] lifecycle_1.0.2        BiocManager_1.30.18    jquerylib_0.1.4       
[100] data.table_1.14.2      bitops_1.0-7           flextable_0.8.1       
[103] httpuv_1.6.6           R6_2.5.1               promises_1.2.0.1      
[106] gridExtra_2.3          writexl_1.4.0          codetools_0.2-18      
[109] MASS_7.3-58.1          assertthat_0.2.1       chron_2.3-57          
[112] rprojroot_2.0.3        withr_2.5.0            GenomeInfoDbData_1.2.8
[115] parallel_4.2.0         hms_1.1.2              grid_4.2.0            
[118] rmarkdown_2.16         googledrive_2.0.0      git2r_0.30.1          
[121] getPass_0.2-2          lubridate_1.8.0        base64enc_0.1-3       </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
