<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="AGC, AY" />


<title>WGCNA</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DEPDC5_D62_Analysis</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">WGCNA</h1>
<h4 class="author">AGC, AY</h4>
<h4 class="date">22 3 2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-08-11
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong>
<code>S:/KJP_Biolabor/Projects/CePTER/DEPDC5_D62_Analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220808code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220808)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220808code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220808)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomKJPMolgenLabDEPDC5D62Analysistreedc78d322307f267fdacb688c7b3be73d2791d9f8targetblankdc78d32a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/KJPMolgenLab/DEPDC5_D62_Analysis/tree/dc78d322307f267fdacb688c7b3be73d2791d9f8" target="_blank">dc78d32</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomKJPMolgenLabDEPDC5D62Analysistreedc78d322307f267fdacb688c7b3be73d2791d9f8targetblankdc78d32a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/KJPMolgenLab/DEPDC5_D62_Analysis/tree/dc78d322307f267fdacb688c7b3be73d2791d9f8" target="_blank">dc78d32</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/

Untracked files:
    Untracked:  output/D62_ResTabsvsNTCnoRAPA_KO_WGCNA.RData
    Untracked:  output/D62_ResTabsvsNTCwRAPA_KO_WGCNA.RData
    Untracked:  output/Restab_D62_NTCnoRAPA_DIFF_RAPA.xlsx
    Untracked:  output/Restab_D62_NTCnoRAPA_DIFF_noRAPA.xlsx
    Untracked:  output/Restab_D62_NTCnoRAPA_noDIFF_RAPA.xlsx
    Untracked:  output/Restab_D62_NTCnoRAPA_noDIFF_noRAPA.xlsx
    Untracked:  output/Restab_D62_NTCwRAPA_DIFF_RAPA.xlsx
    Untracked:  output/Restab_D62_NTCwRAPA_noDIFF_RAPA.xlsx
    Untracked:  output/~$Restab_D62_NTCnoRAPA_DIFF_RAPA.xlsx

Unstaged changes:
    Modified:   analysis/04_D62_WGCNA.Rmd
    Modified:   code/custom_functions.R
    Modified:   output/D62_GOresWGCNA.xlsx
    Modified:   output/D62_ResTabs_KO_WGCNA.RData
    Modified:   output/D62_WGCNA_adj_TOM.RData
    Modified:   output/D62_dds_matrix.RData
    Deleted:    output/Restab_D62_DIFF_RAPA.xlsx
    Deleted:    output/Restab_D62_DIFF_noRAPA.xlsx
    Deleted:    output/Restab_D62_noDIFF_RAPA.xlsx
    Deleted:    output/Restab_D62_noDIFF_noRAPA.xlsx
    Modified:   wflow_helper.R

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with
<code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<pre class="r"><code>home = getwd()</code></pre>
<div id="wgcna" class="section level1">
<h1>WGCNA</h1>
<div id="soft-thresholding" class="section level2">
<h2>soft thresholding</h2>
<pre class="r"><code>allowWGCNAThreads()</code></pre>
<pre><code>Allowing multi-threading with up to 16 threads.</code></pre>
<pre class="r"><code>dds2 = DESeq(ddsMat)</code></pre>
<pre><code>using pre-existing size factors</code></pre>
<pre><code>estimating dispersions</code></pre>
<pre><code>found already estimated dispersions, replacing these</code></pre>
<pre><code>gene-wise dispersion estimates</code></pre>
<pre><code>found already estimated gene-wise dispersions, removing these</code></pre>
<pre><code>mean-dispersion relationship</code></pre>
<pre><code>final dispersion estimates</code></pre>
<pre><code>found already estimated dispersions, removing these</code></pre>
<pre><code>fitting model and testing</code></pre>
<pre class="r"><code>vsd = getVarianceStabilizedData(dds2)

WGCNA_matrix &lt;- t(log2(vsd+1)) #Need to transform for further calculations

#s = abs(bicor(WGCNA_matrix)) #biweight mid-correlation
powers = c(c(1:10), seq(from = 12, to=20, by=2))
sft = pickSoftThreshold(WGCNA_matrix, powerVector = powers, verbose = 5)</code></pre>
<pre><code>pickSoftThreshold: will use block size 3258.
 pickSoftThreshold: calculating connectivity for given powers...
   ..working on genes 1 through 3258 of 13731
   ..working on genes 3259 through 6516 of 13731
   ..working on genes 6517 through 9774 of 13731
   ..working on genes 9775 through 13032 of 13731
   ..working on genes 13033 through 13731 of 13731
   Power SFT.R.sq  slope truncated.R.sq mean.k. median.k. max.k.
1      1    0.234 -0.862          0.909 2910.00  2.82e+03   4940
2      2    0.827 -1.350          0.978  983.00  8.49e+02   2630
3      3    0.935 -1.470          0.984  425.00  3.07e+02   1700
4      4    0.949 -1.490          0.971  216.00  1.24e+02   1230
5      5    0.949 -1.480          0.963  124.00  5.48e+01    945
6      6    0.946 -1.450          0.956   77.20  2.56e+01    759
7      7    0.938 -1.420          0.948   51.50  1.26e+01    627
8      8    0.943 -1.390          0.955   36.10  6.42e+00    530
9      9    0.937 -1.360          0.952   26.40  3.40e+00    456
10    10    0.943 -1.330          0.959   19.90  1.84e+00    397
11    12    0.938 -1.300          0.963   12.20  5.84e-01    311
12    14    0.937 -1.270          0.969    8.08  1.99e-01    251
13    16    0.939 -1.250          0.974    5.62  7.26e-02    207
14    18    0.932 -1.250          0.970    4.06  2.83e-02    174
15    20    0.931 -1.250          0.973    3.03  1.15e-02    148</code></pre>
<pre class="r"><code>par(mfrow = c(1,2))
cex1 = 0.9;
plot(sft$fitIndices[,1], -sign(sft$fitIndices[,3])*sft$fitIndices[,2],
     xlab=&quot;Soft Threshold (power)&quot;,ylab=&quot;Scale Free Topology Model Fit, signed R^2&quot;,
     type=&quot;n&quot;, main = paste(&quot;Scale independence&quot;));
text(sft$fitIndices[,1], -sign(sft$fitIndices[,3])*sft$fitIndices[,2],
     labels=powers,cex=cex1,col=&quot;red&quot;);
abline(h=0.80,col=&quot;red&quot;)

plot(sft$fitIndices[,1], sft$fitIndices[,5],xlab=&quot;Soft Threshold (power)&quot;,ylab=&quot;Mean Connectivity&quot;, type=&quot;n&quot;,main = paste(&quot;Mean connectivity&quot;))
text(sft$fitIndices[,1], sft$fitIndices[,5], labels=powers, cex=cex1,col=&quot;red&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/softthreshold-1.png" width="4500" style="display: block; margin: auto;" /></p>
</div>
<div id="identify-gene-modules" class="section level2">
<h2>Identify Gene Modules</h2>
<pre class="r"><code>softPower = 6;

# The point where the curve flattens
#calclute the adjacency matrix

if(reanalyze | !file.exists(paste0(output,&quot;/D62_WGCNA_adj_TOM.RData&quot;))){
  #adj= adjacency(WGCNA_matrix,type = &quot;unsigned&quot;, power = softPower);
  
  #Converting adjacency matrix into To so that the noise could be reduced
  TOM=TOMsimilarityFromExpr(WGCNA_matrix,networkType = &quot;unsigned&quot;, 
                            TOMType = &quot;unsigned&quot;, power = softPower);
  save(list = c(
    #&quot;adj&quot;, 
    &quot;TOM&quot;), file=paste0(output,&quot;/D62_WGCNA_adj_TOM.RData&quot;))
  
} else {
  load(paste0(output,&quot;/D62_WGCNA_adj_TOM.RData&quot;))
}


SubGeneNames&lt;-colnames(WGCNA_matrix)

colnames(TOM) =rownames(TOM) =SubGeneNames
dissTOM=1-TOM
diag(dissTOM) = 0

#hierarchical clustering
geneTree = flashClust(as.dist(dissTOM),method=&quot;average&quot;);

#plot the resulting clustering tree (dendrogram)
# plot(geneTree, xlab=&quot;&quot;, sub=&quot;&quot;,cex=0.3, main=&quot;Module clustering prio to merging&quot;);

#Set the minimum module size
minModuleSize = 50;

#Module identification using dynamic tree cut
dynamicMods = cutreeDynamic(dendro = geneTree, 
                            distM = dissTOM, 
                            cutHeight = 0.998,
                            minClusterSize = minModuleSize,
                            deepSplit=2, 
                            pamRespectsDendro = T)</code></pre>
<pre><code> ..done.</code></pre>
<pre class="r"><code>#the following command gives the module labels and the size of each module. Lable 0 is reserved for unassigned genes
#table(dynamicMods)

#Plot the module assignment under the dendrogram; note: The grey color is reserved for unassigned genes
dynamicColors = labels2colors(dynamicMods)
table(dynamicColors)</code></pre>
<pre><code>dynamicColors
    black      blue     brown     green      grey       red turquoise    yellow 
      276      2306      1716      1024       404       658      6000      1347 </code></pre>
<pre class="r"><code>plotDendroAndColors(geneTree, dynamicColors, &quot;Dynamic Tree Cut&quot;, 
                    dendroLabels = FALSE, hang = 0.03, 
                    addGuide = TRUE, 
                    guideHang = 0.05, 
                    main = &quot;Gene dendrogram and module colors&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/adjacency-1.png" width="4500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>diag(dissTOM) = NA;</code></pre>
<pre class="r"><code>#Visualize the Tom plot. Raise the dissimilarity matrix to the power of 4 to bring out the module structure
TOMplot(dissTOM^4, geneTree, as.character(dynamicColors), main=&quot;weighted distance of Topological overlap Matrix&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/adjacency_TOMplot-1.png" width="4500" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="module-eigengenes" class="section level1">
<h1>Module eigengenes</h1>
<div id="identify-and-merge-correlated-modules" class="section level2">
<h2>Identify and Merge correlated modules</h2>
<pre class="r"><code>#calculate eigengenes
MEList = moduleEigengenes(WGCNA_matrix, colors = dynamicColors)
MEs = MEList$eigengenes 

plotEigengeneNetworks(MEs, &quot;Eigengenes_before_merging&quot;, 
                      marDendro = c(0,4,1,2), marHeatmap = c(3,4,1,2))</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/MEs-1.png" width="4000" style="display: block; margin: auto;" /></p>
<pre class="r"><code>MEList_new = mergeCloseModules(WGCNA_matrix, colors = dynamicColors, MEs = MEs, cutHeight = 0.05)</code></pre>
<pre><code> mergeCloseModules: Merging modules whose distance is less than 0.05
   Calculating new MEs...</code></pre>
<pre class="r"><code>plotEigengeneNetworks(MEList_new$newMEs, &quot;Eigengenes_after_merging&quot;, 
                      marDendro = c(0,4,1,2), marHeatmap = c(3,4,1,2))</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/MEs-2.png" width="4000" style="display: block; margin: auto;" /></p>
<pre class="r"><code>MEs = MEList_new$newMEs

coldata_all = colData(ddsMat) %&gt;% as.data.frame() %&gt;% 
  select(-grep(&quot;ME&quot;, colnames(colData(ddsMat)), value = T)) %&gt;% 
  cbind(MEs[colnames(ddsMat),]) %&gt;% DataFrame()

colData(ddsMat) = coldata_all

colors_new = MEList_new$colors

mcols(ddsMat) = cbind(mcols(ddsMat)%&gt;% as.data.frame() %&gt;% select(-contains(&quot;cluster&quot;)), data.frame(cluster=colors_new)) %&gt;% DataFrame()


save(ddsMat, file=paste0(output,&quot;/D62_dds_matrix.RData&quot;))</code></pre>
</div>
<div id="heatmaps-mes" class="section level2">
<h2>Heatmaps MEs</h2>
<pre class="r"><code>#colors for plotting heatmap
colors &lt;- rev(colorRampPalette(brewer.pal(9, &quot;Spectral&quot;))(255))


gRNAcol = Dark8[c(1:nlevels(SampleInfo$gRNA))+nlevels(SampleInfo$CellLine)]
names(gRNAcol) = levels(SampleInfo$gRNA)

diffcol = brewer.pal(3,&quot;Set1&quot;)[1:nlevels(SampleInfo$DIFF)]
names(diffcol) = levels(SampleInfo$DIFF)

rapacol = brewer.pal(3,&quot;Set2&quot;)[1:nlevels(SampleInfo$RAPA)]
names(rapacol) = levels(SampleInfo$RAPA)

clustcol = gplots::col2hex(unique(as.character(mcols(ddsMat)$cluster)))
names(clustcol) = unique(as.character(mcols(ddsMat)$cluster))

rownames(WGCNA_matrix)=SampleInfo[rownames(WGCNA_matrix), &quot;label_rep&quot;]

ann_colors = list(
  DIFF = diffcol, 
  RAPA = rapacol,
  gRNA = gRNAcol,
  cluster = clustcol)

idx=order(SampleInfo$gRNA, SampleInfo$DIFF,SampleInfo$RAPA)
WGCNA_matrix_sorted=WGCNA_matrix[SampleInfo$label_rep[idx], order(colors_new)]

collabels = SampleInfo[idx,c(&quot;gRNA&quot;,&quot;DIFF&quot;, &quot;RAPA&quot;)] %&gt;%  
  mutate_all(as.character) %&gt;% as.data.frame()

rownames(collabels)=SampleInfo$label_rep[idx]

genlabels = data.frame(cluster = as.character(colors_new)[order(colors_new)])
rownames(genlabels) = colnames(WGCNA_matrix_sorted)


MElabels = data.frame(cluster = gsub(&quot;ME&quot;, &quot;&quot;,colnames(MEs)))
rownames(MElabels) = colnames(MEs)

clustcol = gplots::col2hex(unique(as.character(MElabels$cluster)))
names(clustcol) = as.character(MElabels$cluster)

ann_colors = list(
  DIFF = diffcol, 
  RAPA = rapacol,
  gRNA = gRNAcol,
  cluster = clustcol)

rownames(MEs) = SampleInfo[rownames(MEs),&quot;label_rep&quot;]

pheatmap(t(MEs[idx,]),
         border_color = NA, 
         annotation_row = MElabels,
         annotation_col = collabels,
         cluster_cols = F,
         show_rownames = F, show_colnames = F,
         clustering_method = &quot;ward.D2&quot;,
         annotation_colors = ann_colors,
         scale=&quot;row&quot;,
         breaks  = seq(-2, 2,length.out=255),
         col = colors, 
         main = &quot;eigengene values&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/heatmap-1.png" width="2500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>SampleInfo = as.data.frame(colData(ddsMat))
MEMat = SampleInfo[,grep(&quot;ME&quot;, colnames(SampleInfo))]



## helper functions#test differences
testit=function(Dataset, samples = Set, 
                depvar){
  data = Dataset[samples,]
  res=list()
  for(i in grep(&quot;ME&quot;, colnames(data), value = T)){
    res[[i]]=lm(as.formula(paste0(i,&quot;~1+&quot;,depvar)), data)
  }
  return(res)
}



# extract coefficents
getcoefff=function(x){
  res = summary(x)$coefficients[2,]
  return(res)
}

# comparisonME
comparisonME = function(SampleInfo, Set, target){
  LMlist=testit(Dataset = SampleInfo, samples = Set, depvar=target)
  coeff = as.data.frame(lapply(LMlist, getcoefff) %&gt;% do.call(rbind, .))
  coeff$padj = p.adjust(coeff$`Pr(&gt;|t|)`, &quot;bonferroni&quot;)
  return(coeff)}

## comparisons against noRAPA NTC
Rapamycin=c(&quot;noRAPA&quot;, &quot;RAPA&quot;)
Differentiation=c(&quot;noDIFF&quot;, &quot;DIFF&quot;)
Type_sgRNA&lt;-c(&quot;sg2.1&quot;,&quot;sg2.2&quot;)
target=&quot;KO&quot;

r = Rapamycin[1]
d = Differentiation[1]
Tp = Type_sgRNA[[1]]


# no random effects included 
for(r in Rapamycin){
  Rapafilter = SampleInfo$RAPA %in% r
  for(d in Differentiation){
    Difffilter = SampleInfo$DIFF %in% d
    for(Tp in Type_sgRNA){
      sgRNAfilter = SampleInfo$gRNA %in% Tp
      vs_label=paste0(c(&quot;sgNTC&quot;, Tp), sep=&quot;&quot;, collapse=&quot;_&quot;)
        Set = rownames(SampleInfo)[Rapafilter&amp;Difffilter&amp;
                                     sgRNAfilter]
        Set = c(Set, rownames(SampleInfo)[SampleInfo$RAPA == &quot;noRAPA&quot; &amp; Difffilter&amp;
                                     SampleInfo$gRNA == &quot;sgNTC&quot;])
        lab = paste(&quot;restabWGCNA&quot;, &quot;D62_NTCnoRAPA&quot;, vs_label, d,r, sep=&quot;_&quot;)
        assign(lab, comparisonME(SampleInfo, Set, target))
    }
  }
}

comparisons= apropos(&quot;restabWGCNA_D62_NTCnoRAPA&quot;)
save(list = comparisons, file = paste0(home,&quot;/output/D62_ResTabsvsNTCnoRAPA_KO_WGCNA.RData&quot;))


# comparison agains RAPANTC
Rapamycin=c(&quot;noRAPA&quot;, &quot;RAPA&quot;)
Differentiation=c(&quot;noDIFF&quot;, &quot;DIFF&quot;)
Type_sgRNA&lt;-list(c(&quot;sgNTC&quot;,&quot;sg2.1&quot;),c(&quot;sgNTC&quot;, &quot;sg2.2&quot;))
target=&quot;KO&quot;

# comparison agains RAPANTC
for(r in Rapamycin){
  Rapafilter = SampleInfo$RAPA %in% r
  for(d in Differentiation){
    Difffilter = SampleInfo$DIFF %in% d
    for(Tp in Type_sgRNA){
      sgRNAfilter = SampleInfo$gRNA %in% Tp
      vs_label=paste0(Tp, sep=&quot;&quot;, collapse=&quot;_&quot;)
        Set = rownames(SampleInfo)[Rapafilter&amp;Difffilter&amp;
                                     sgRNAfilter]
        lab = paste(&quot;restabWGCNA&quot;, &quot;D62_NTCwRAPA&quot;, vs_label, d,r, sep=&quot;_&quot;)
        assign(lab, comparisonME(SampleInfo, Set, target))
    }
  }
}

comparisons= apropos(&quot;restabWGCNA_D62_NTCwRAPA&quot;)
save(list = comparisons, file = paste0(home,&quot;/output/D62_ResTabsvsNTCwRAPA_KO_WGCNA.RData&quot;))</code></pre>
<pre class="r"><code>mypval=0.05
MEIds = rownames(get(apropos(&quot;restabWGCNA&quot;)[1]))

getWGCNAoutputs = function(targetline=&quot;D62&quot;,targetdiff,targetrapa, refset=&quot;NTCnoRAPA&quot;, plotset=F){
  
  samplesincl = SampleInfo$DIFF==targetdiff &amp; 
    SampleInfo$RAPA==targetrapa
  
  pvalrep=  get(paste0(&quot;restabWGCNA_&quot;, 
                       targetline,&quot;_&quot;,refset, &quot;_sgNTC_sg2.1_&quot;, 
                       targetdiff, &quot;_&quot; , 
                       targetrapa))$padj&lt;=mypval &amp;
    get(paste0(&quot;restabWGCNA_&quot;, 
               targetline, &quot;_&quot;,refset, &quot;_sgNTC_sg2.2_&quot;, 
               targetdiff, &quot;_&quot; , 
               targetrapa))$padj&lt;=mypval
  
  betarep = apply(cbind(get(paste0(&quot;restabWGCNA_&quot;, 
                                   targetline, &quot;_&quot;,refset, &quot;_sgNTC_sg2.1_&quot;, 
                                   targetdiff, &quot;_&quot; , 
                                   targetrapa))$Estimate,
                        get(paste0(&quot;restabWGCNA_&quot;, 
                                   targetline, &quot;_&quot;,refset, &quot;_sgNTC_sg2.2_&quot;, 
                                   targetdiff, &quot;_&quot; , 
                                   targetrapa))$Estimate), 1,
                  samesign)
  
  idx=which(betarep &amp; pvalrep)
  hits=MEIds[idx]
  
  
  restab=data.frame(
    Module = MEIds, 
    beta_2.1 = get(paste0(&quot;restabWGCNA_&quot;, 
                          targetline,&quot;_&quot;,refset,  &quot;_sgNTC_sg2.1_&quot;, 
                          targetdiff, &quot;_&quot; , 
                          targetrapa))$Estimate, 
    bonferroni_2.1 = get(paste0(&quot;restabWGCNA_&quot;, 
                                targetline, &quot;_&quot;,refset, &quot;_sgNTC_sg2.1_&quot;, 
                                targetdiff, &quot;_&quot; , 
                                targetrapa))$padj, 
    beta_2.2 = get(paste0(&quot;restabWGCNA_&quot;, 
                          targetline,&quot;_&quot;,refset,  &quot;_sgNTC_sg2.2_&quot;, 
                          targetdiff, &quot;_&quot; , 
                          targetrapa))$Estimate, 
    bonferroni_2.2 = get(paste0(&quot;restabWGCNA_&quot;, 
                                targetline, &quot;_&quot;,refset, &quot;_sgNTC_sg2.2_&quot;, 
                                targetdiff, &quot;_&quot; , 
                                targetrapa))$padj)
  
  print(restab)
  write.xlsx(restab, file=paste0(output, &quot;/Restab_&quot;, 
                                 targetline,&quot;_&quot;,refset, &quot;_&quot;, 
                                 targetdiff, &quot;_&quot;,
                                 targetrapa, &quot;.xlsx&quot;))
  SamplesSet=SampleInfo[samplesincl,] %&gt;% select(all_of(hits))
  if(plotset){
  EigengenePlot(SamplesSet, SampleInfo, samplesincl)}
  
}</code></pre>
</div>
<div id="eigengene-plots-all-modules" class="section level2">
<h2>Eigengene plots all modules</h2>
<pre class="r"><code>EigengenePlot(data=SampleInfo[,grep(&quot;ME&quot;, colnames(SampleInfo))],
              Sampledata = SampleInfo, 
              samplesincl=rep(T, nrow(SampleInfo)))</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-1.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-2.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-3.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-4.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-5.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-6.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-7.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-2-8.png" width="4500" style="display: block; margin: auto;" /></p>
</div>
<div id="wgcna-module-plots-only-d62" class="section level2">
<h2>WGCNA module plots only D62</h2>
</div>
<div id="ko-effect-in-nodiff-norapa" class="section level2">
<h2>KO effect in noDIFF noRAPA</h2>
<div id="d62" class="section level3">
<h3>D62</h3>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;noDIFF&quot;,targetrapa = &quot;noRAPA&quot;, plotset = T)</code></pre>
<pre><code>       Module    beta_2.1 bonferroni_2.1     beta_2.2 bonferroni_2.2
1       MEred  0.34049833   0.0009947568 -0.009847209   1.000000e+00
2     MEbrown -0.28206000   0.0003735108 -0.058385040   2.518703e-02
3      MEblue -0.33523842   0.0022606324  0.002375805   1.000000e+00
4     MEgreen -0.32410775   0.0022396816 -0.008367778   1.000000e+00
5     MEblack -0.11039767   0.0079997639 -0.069518536   7.682637e-05
6 MEturquoise  0.01696263   0.3861421012  0.004109875   1.101362e-01
7    MEyellow -0.29617673   0.0084866670 -0.005792508   1.000000e+00
8      MEgrey -0.18704459   1.0000000000 -0.040649469   1.000000e+00</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/noDIFF_noRAPA_D62-1.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/noDIFF_noRAPA_D62-2.png" width="4500" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ko-effect-in-diff-norapa" class="section level2">
<h2>KO effect in DIFF noRAPA</h2>
<div id="d62-1" class="section level3">
<h3>D62</h3>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;DIFF&quot;,targetrapa = &quot;noRAPA&quot;, plotset = T)</code></pre>
<pre><code>       Module    beta_2.1 bonferroni_2.1    beta_2.2 bonferroni_2.2
1       MEred -0.31000479   0.0005785977 -0.01738949   1.000000e+00
2     MEbrown  0.28494983   0.0009531967  0.26751850   1.186855e-03
3      MEblue  0.35131310   0.0004080748  0.05345781   9.457238e-01
4     MEgreen  0.33612032   0.0016021205  0.06979599   9.492983e-01
5     MEblack  0.24588372   0.0018629558  0.51251646   5.187381e-05
6 MEturquoise -0.03377917   0.1895466925 -0.04773828   3.723205e-02
7    MEyellow  0.24513519   0.0018442002  0.03527412   5.519585e-01
8      MEgrey -0.14601285   1.0000000000 -0.11812970   1.000000e+00</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/DIFF_noRAPA_D62-1.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/DIFF_noRAPA_D62-2.png" width="4500" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="ko-effect-in-nodiff-rapa" class="section level2">
<h2>KO effect in noDIFF RAPA</h2>
<div id="d62-2" class="section level3">
<h3>D62</h3>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;noDIFF&quot;,targetrapa = &quot;RAPA&quot;, refset = &quot;NTCwRAPA&quot; , plotset = T)</code></pre>
<pre><code>       Module     beta_2.1 bonferroni_2.1     beta_2.2 bonferroni_2.2
1       MEred  0.063701420    1.000000000 -0.007754679     1.00000000
2     MEbrown -0.082873602    1.000000000 -0.028123938     1.00000000
3      MEblue -0.073531605    1.000000000  0.038619733     1.00000000
4     MEgreen -0.071292095    1.000000000  0.029769245     1.00000000
5     MEblack -0.094069613    0.006916798 -0.060935174     0.02118051
6 MEturquoise  0.005791482    1.000000000 -0.006929335     1.00000000
7    MEyellow -0.050955126    1.000000000 -0.013454312     1.00000000
8      MEgrey -0.120824426    0.609904240 -0.107417138     1.00000000</code></pre>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;noDIFF&quot;,targetrapa = &quot;RAPA&quot;, refset = &quot;NTCnoRAPA&quot; , plotset = T)</code></pre>
<pre><code>       Module     beta_2.1 bonferroni_2.1     beta_2.2 bonferroni_2.2
1       MEred  0.172550699     0.91485849  0.101094599     1.00000000
2     MEbrown  0.062588843     1.00000000  0.117338508     0.39737331
3      MEblue -0.145985884     1.00000000 -0.033834546     1.00000000
4     MEgreen -0.108069293     1.00000000 -0.007007953     1.00000000
5     MEblack -0.058527235     0.03022928 -0.025392796     0.28882806
6 MEturquoise -0.003593694     1.00000000 -0.016314511     0.04912674
7    MEyellow -0.161668997     0.38875014 -0.124168183     1.00000000
8      MEgrey -0.137741921     0.84785240 -0.124334633     1.00000000</code></pre>
</div>
</div>
<div id="ko-effect-in-diff-rapa" class="section level2">
<h2>KO effect in DIFF RAPA</h2>
<div id="d62-3" class="section level3">
<h3>D62</h3>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;DIFF&quot;,targetrapa = &quot;RAPA&quot;, refset = &quot;NTCwRAPA&quot; , plotset = T)</code></pre>
<pre><code>       Module    beta_2.1 bonferroni_2.1    beta_2.2 bonferroni_2.2
1       MEred -0.02196168    1.000000000 -0.04280098     1.00000000
2     MEbrown  0.06846119    1.000000000  0.14037625     1.00000000
3      MEblue  0.04906314    1.000000000  0.08871593     1.00000000
4     MEgreen  0.03845811    1.000000000  0.01787810     1.00000000
5     MEblack  0.20084728    0.003557251  0.27884853     0.01207385
6 MEturquoise  0.02340289    0.799383362  0.01486376     1.00000000
7    MEyellow  0.00412049    1.000000000  0.04545928     1.00000000
8      MEgrey -0.13340904    0.743595737 -0.18919134     0.58800950</code></pre>
<pre class="r"><code>getWGCNAoutputs(targetdiff = &quot;DIFF&quot;,targetrapa = &quot;RAPA&quot;, refset = &quot;NTCnoRAPA&quot; , plotset = T)</code></pre>
<pre><code>       Module     beta_2.1 bonferroni_2.1     beta_2.2 bonferroni_2.2
1       MEred -0.294763402      0.2542855 -0.315602699      0.4472014
2     MEbrown  0.100193723      1.0000000  0.172108789      0.5446000
3      MEblue  0.291478418      0.3637644  0.331131204      0.6093049
4     MEgreen  0.202459592      0.7402675  0.181879584      1.0000000
5     MEblack  0.047129344      0.6176020  0.125130599      0.2148778
6 MEturquoise  0.003062091      1.0000000 -0.005477041      1.0000000
7    MEyellow  0.277153407      0.1630285  0.318492193      0.1866296
8      MEgrey -0.240546841      1.0000000 -0.296329138      0.9021750</code></pre>
<pre><code>Warning in file.create(to[okay]): kann Datei &#39;S:/KJP_Biolabor/Projects/CePTER/
DEPDC5_D62_Analysis/output//Restab_D62_NTCnoRAPA_DIFF_RAPA.xlsx&#39; nicht erzeugen.
Grund &#39;Permission denied&#39;</code></pre>
</div>
</div>
<div id="ko-additional-plots-for-better-visulization"
class="section level2">
<h2>KO additional Plots for better visulization</h2>
<pre class="r"><code>dataset=SampleInfo[,grep(&quot;brown|black&quot;, colnames(SampleInfo))]
idx=order(SampleInfo$RAPA, SampleInfo$gRNA)
SampleInfosort = SampleInfo[idx,]

EigengenePlot(data=dataset,
              Sampledata = SampleInfosort, 
              samplesincl=SampleInfosort$DIFF==&quot;noDIFF&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-3-1.png" width="4500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>EigengenePlot(data=dataset,
              Sampledata = SampleInfosort, 
              samplesincl=SampleInfosort$DIFF==&quot;DIFF&quot;)</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-3-2.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-3-3.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/unnamed-chunk-3-4.png" width="4500" style="display: block; margin: auto;" /></p>
<pre class="r"><code>gene_univers = rownames(ddsMat)

Genes_of_interset = split(rownames(ddsMat), mcols(ddsMat)$cluster)

gostres = getGOresults(Genes_of_interset, gene_univers)

toptab = gostres$result

write.xlsx(toptab, file = paste0(output,&quot;/D62_GOresWGCNA.xlsx&quot;), sheetName = &quot;GO_enrichment&quot;)

for (module in names(Genes_of_interset)){
  idx = toptab$query==module &amp; grepl(&quot;GO&quot;, toptab$source)
  if(!any(idx)){
    p = ggplot() + annotate(&quot;text&quot;, x = 4, y = 25, size=4, 
                            label = &quot;no significant GO term&quot;) + 
      ggtitle(module)+theme_void()+
      theme(plot.title = element_text(hjust = 0.5))
  } else {
    p=GOplot(toptab[idx, ], 10, Title =module)
  }
  print(p)
}</code></pre>
<p><img src="figure/04_D62_WGCNA.Rmd/goresplots-1.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-2.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-3.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-4.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-5.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-6.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-7.png" width="4500" style="display: block; margin: auto;" /><img src="figure/04_D62_WGCNA.Rmd/goresplots-8.png" width="4500" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.0 (2022-04-22 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.utf8  LC_CTYPE=German_Germany.utf8   
[3] LC_MONETARY=German_Germany.utf8 LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.utf8    

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] openxlsx_4.2.5              gprofiler2_0.2.1           
 [3] flashClust_1.01-2           WGCNA_1.71                 
 [5] fastcluster_1.2.3           dynamicTreeCut_1.63-1      
 [7] knitr_1.39                  DESeq2_1.36.0              
 [9] SummarizedExperiment_1.26.1 Biobase_2.56.0             
[11] MatrixGenerics_1.8.0        matrixStats_0.62.0         
[13] GenomicRanges_1.48.0        GenomeInfoDb_1.32.2        
[15] IRanges_2.30.0              S4Vectors_0.34.0           
[17] BiocGenerics_0.42.0         pheatmap_1.0.12            
[19] RColorBrewer_1.1-3          compareGroups_4.5.1        
[21] forcats_0.5.1               stringr_1.4.0              
[23] dplyr_1.0.9                 purrr_0.3.4                
[25] readr_2.1.2                 tidyr_1.2.0                
[27] tibble_3.1.7                ggplot2_3.3.6              
[29] tidyverse_1.3.1             kableExtra_1.3.4           

loaded via a namespace (and not attached):
  [1] readxl_1.4.0           uuid_1.1-0             backports_1.4.1       
  [4] Hmisc_4.7-0            workflowr_1.7.0        systemfonts_1.0.4     
  [7] lazyeval_0.2.2         splines_4.2.0          BiocParallel_1.30.3   
 [10] digest_0.6.29          foreach_1.5.2          htmltools_0.5.2       
 [13] GO.db_3.15.0           fansi_1.0.3            checkmate_2.1.0       
 [16] magrittr_2.0.3         Rsolnp_1.16            memoise_2.0.1         
 [19] cluster_2.1.3          doParallel_1.0.17      tzdb_0.3.0            
 [22] Biostrings_2.64.0      annotate_1.74.0        modelr_0.1.8          
 [25] officer_0.4.3          svglite_2.1.0          jpeg_0.1-9            
 [28] colorspace_2.0-3       blob_1.2.3             rvest_1.0.2           
 [31] haven_2.5.0            xfun_0.31              crayon_1.5.1          
 [34] RCurl_1.98-1.6         jsonlite_1.8.0         genefilter_1.78.0     
 [37] impute_1.70.0          survival_3.3-1         iterators_1.0.14      
 [40] glue_1.6.2             gtable_0.3.0           zlibbioc_1.42.0       
 [43] XVector_0.36.0         webshot_0.5.3          DelayedArray_0.22.0   
 [46] scales_1.2.0           DBI_1.1.3              Rcpp_1.0.9            
 [49] htmlTable_2.4.1        viridisLite_0.4.0      xtable_1.8-4          
 [52] foreign_0.8-82         bit_4.0.4              preprocessCore_1.58.0 
 [55] Formula_1.2-4          truncnorm_1.0-8        htmlwidgets_1.5.4     
 [58] httr_1.4.3             gplots_3.1.3           ellipsis_0.3.2        
 [61] mice_3.14.0            farver_2.1.1           pkgconfig_2.0.3       
 [64] XML_3.99-0.10          deldir_1.0-6           nnet_7.3-17           
 [67] sass_0.4.1             dbplyr_2.2.1           locfit_1.5-9.6        
 [70] utf8_1.2.2             labeling_0.4.2         tidyselect_1.1.2      
 [73] rlang_1.0.2            later_1.3.0            AnnotationDbi_1.58.0  
 [76] munsell_0.5.0          cellranger_1.1.0       tools_4.2.0           
 [79] cachem_1.0.6           cli_3.3.0              generics_0.1.3        
 [82] RSQLite_2.2.14         broom_1.0.0            evaluate_0.15         
 [85] fastmap_1.1.0          yaml_2.3.5             bit64_4.0.5           
 [88] fs_1.5.2               zip_2.2.0              caTools_1.18.2        
 [91] KEGGREST_1.36.3        xml2_1.3.3             compiler_4.2.0        
 [94] rstudioapi_0.13        plotly_4.10.0          png_0.1-7             
 [97] reprex_2.0.1           geneplotter_1.74.0     bslib_0.4.0           
[100] stringi_1.7.6          HardyWeinberg_1.7.5    highr_0.9             
[103] gdtools_0.2.4          lattice_0.20-45        Matrix_1.4-1          
[106] vctrs_0.4.1            pillar_1.8.0           lifecycle_1.0.1       
[109] jquerylib_0.1.4        data.table_1.14.2      bitops_1.0-7          
[112] flextable_0.7.2        httpuv_1.6.5           latticeExtra_0.6-30   
[115] R6_2.5.1               promises_1.2.0.1       KernSmooth_2.23-20    
[118] gridExtra_2.3          writexl_1.4.0          codetools_0.2-18      
[121] gtools_3.9.3           assertthat_0.2.1       chron_2.3-57          
[124] rprojroot_2.0.3        withr_2.5.0            GenomeInfoDbData_1.2.8
[127] parallel_4.2.0         hms_1.1.1              rpart_4.1.16          
[130] grid_4.2.0             rmarkdown_2.14         git2r_0.30.1          
[133] lubridate_1.8.0        base64enc_0.1-3        interp_1.1-3          </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
